test/Unit/Queue/TimeLimitedQueueTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Genkgo\TestMail\Unit\Queue;
+
+use DateTime;
```

Comment:
I believe I never import global namespaced objects.

Reply:
Ah, I copied what you had done in `QueueIfFailedTransport`
https://github.com/genkgo/mail/blob/db576bbcd6d75457c10199df6294bd12a49ed658/src/Transport/QueueIfFailedTransport.php#L6

I'll remove mine, would you like me to get the one in `QueueIfFailedTransport` too?

Reply:
Yes, I don't what is the best practise in this regard, but let's apply the same method everywhere. So yes, if you would like to change that, please do.

Reply:
Done.

> I don't what is the best practise in this regard

Yeah, I go for importing everything so that it is visible from just the import list what is in use in the class. 

>  but let's apply the same method everywhere

Yes, as I was trying to follow what was in the existing code (albeit what I was directly looking at) for reference. Also one reason for the CS PR, the overall consistency … and a way for a contributor to automatically apply a lot of it.